clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/BaseIntegrationSpec.scala

```diff
@@ -282,6 +282,17 @@ abstract class BaseIntegrationSpec(clioDescription: String)
       .write(obj.asJson.pretty(implicitly[Printer]))
   }
 
+  /**
+    * Validate that a String is valid JSON and write it
+    * to a local temp file if it is. Throw an exception if it isn't.
```

Comment:
I don't think this actually throws an exception if the json is invalid.

Reply:
Ah, yeah. It used to. Then I decided I didn't want it to. Will update comment